### ISSUE TITLE
Getting "Error: no such option: --include *.txt"

### ISSUE THREAD

**Issue by whataride** [Azure/batch-shipyard#212]
I am doing something fairly simple, which worked before:

![image](https://user-images.githubusercontent.com/6028927/41261480-bf466df4-6d90-11e8-83c2-59f202de1af5.png)

And I got this error when submitting it to Azure Batch for execution:

![image](https://user-images.githubusercontent.com/6028927/41261521-e056b008-6d90-11e8-9f70-50d0594ff89e.png)

I looked around. It seems that the --include option is specified for `shipyard_blobxfer.sh`

```
$AZ_BATCH_NODE_STARTUP_DIR/wd/shipyard_blobxfer.sh "1.2.0,i,False,batchdev,core.windows.net,sr=c&sp=rl&sig=XXX&sv=2017-07-29&se=2048-06-03T22%3A18%3A42Z,ingressed-files/test,$AZ_BATCH_TASK_WORKING_DIR,--include "*.txt" "
```

I further looked `wd/shipyard_blobxfer.sh`, but I don't see where the `--include` is used. So I am not sure if I looked at the right place.

**Comment by alfpark** [comment]
I believe this was a regression that has been recently fixed. Can you see if the `develop` branch or the `develop-cli` docker image fixes your issue?

**Comment by alfpark** [comment]
This issue should be fixed in `3.5.0b2`. Please re-open if you are still having issues.

**Comment by whataride** [comment]
Yes, the issue is fixed. Thanks for the quick turnaround!
